internal/alertmanager/currentalert/process.go

```diff
@@ -141,3 +164,10 @@ func upScale(labels model.LabelSet, annotations model.LabelSet, client client.Cl
 	}
 	return nil
 }
+
+func (c *ccConfig) getCruiseControlStatus() error {
+	if err := scale.GetCruiseControlStatus(c.Namespace, c.CruiseControlEndpoint, c.Name); err != nil {
```

Comment:
This can be simplified to `return scale.GetCruiseControlStatus(c.Namespace, c.CruiseControlEndpoint, c.Name)`